### PR TITLE
community: Integration for `Friendli` LLM and `ChatFriendli` ChatModel.

### PR DESCRIPTION
**Pull request by kooyunmo**
## Description
- Add [Friendli](https://friendli.ai/) integration for `Friendli` LLM and `ChatFriendli` chat model.
- Unit tests and integration tests corresponding to this change are added.
- Documentations corresponding to this change are added.

## Dependencies
- Optional dependency [`friendli-client`](https://pypi.org/project/friendli-client/) package is added only for those who use `Frienldi` or `ChatFriendli` model.

## Twitter handle
- https://twitter.com/friendliai

### CODE REVIEW

File: libs/community/tests/integration_tests/chat_models/test_friendli.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+from langchain_community.chat_models.friendli import ChatFriendli
+
+
+@pytest.fixture
+def friendli_chat() -> ChatFriendli:
+    """Friendli LLM for chat."""
+    return ChatFriendli(temperature=0, max_tokens=10)
+
+
+@pytest.mark.scheduled
```
Comment (baskaryan) [inline libs/community/tests/integration_tests/chat_models/test_friendli.py:18]: @kooyunmo scheduled testing requires us to have a Friendli token set up in the repo secrets. happy to set that up if you'd like to send us a token, can send to bagatur@langchain.dev. but don't think we should block this PR on that so for now would recommend removing pytest.mark.scheduled

Comment (kooyunmo) [reply libs/community/tests/integration_tests/chat_models/test_friendli.py]: @baskaryan Thanks for the quick review! I removed `pytest.mark.scheduled` from all integrated test cases I added as you suggested.

**Comment by kooyunmo** [comment]
@baskaryan Could you please check this PR when you are available? It seems that CI does not run for a few days.

**Comment by kooyunmo** [comment]
@baskaryan I've tried to update the branch of this PR whenever the master branch is updated, but Actions does not run sometimes. Do I need to keep the branch updated to get merged?

**Comment by kooyunmo** [comment]
@efriis @eyurtsev @hwchase17 I am sorry for mentioning you all, but could you kindly review this pull request in case @baskaryan is currently unavailable? I've been just syncing up this branch with `master` for the past few days 🥲

**Review by baskaryan — APPROVED** [review]

**Comment by kooyunmo** [comment]
@baskaryan The conflicts at `pyproject.toml` and `poetry.lock` are resolved.

**Comment by kooyunmo** [comment]
@baskaryan @efriis @eyurtsev @hwchase17 Can we merge this approved PR at the moment?

**Comment by kooyunmo** [comment]
@baskaryan I am sorry for bothering you again but can you please check if this PR is good to merge?

**Pull request merged**